omero_marshal/decode/__init__.py

```diff
@@ -34,7 +35,8 @@ def to_unit(self, v):
 
     def set_property(self, target, prop, value):
         field_info = getattr(target._field_info, prop)
-        if isinstance(value, unicode):
+        # When would we NOT want to do this?
```

Comment:
There's at least one other example of this, but I'll have to dig it up. I think this is a case where one differentiates between py2/py3.